samples/crypto/kmu_usage_nrf54l/README.rst

```diff
@@ -19,7 +19,7 @@ The sample supports the following development kits:
 
```

Comment:
- Edit line 3 (title) to `Crypto: KMU usage`.
- Edit line 10 to `The KMU usage sample demonstrates how to generate cryptographic keys for nRF54L Series devices and nRF7120 and securely store them in the Key Management Unit (KMU).`
- Edit line 30 to `These keys are stored in the :ref:`Key Management Unit (KMU) <ug_nrf54l_developing_basics_kmu>` of the device and retain their value between device resets.`
- Edit line 31 to `The KMU is one of the :ref:`key storage <key_storage>` options in the |NCS|, available only on the nRF54L Series and nRF7120 devices.